Index: backend/.env.example
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/backend/.env.example b/backend/.env.example
new file mode 100644
--- /dev/null	(date 1768485754082)
+++ b/backend/.env.example	(date 1768485754082)
@@ -0,0 +1,6 @@
+DB_NAME=Node
+DB_USER=postgres
+DB_PASSWORD=qwerty123
+DB_HOST=localhost
+DB_PORT=5432
+JWT_SECRET=secret
