Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
deleted file mode 100644
--- a/.env	(revision 7c98a701264a0f76f716cdab6c5c74120c702058)
+++ /dev/null	(revision 7c98a701264a0f76f716cdab6c5c74120c702058)
@@ -1,6 +0,0 @@
-PGHOST=localhost
-PGPORT=5432
-PGUSER=postgres
-PGPASSWORD=qwerty123
-PGDATABASE=Node
-PORT=3000
